src/applications/vaos/covid-19-vaccine/components/ConfirmationPage.jsx

```diff
@@ -68,9 +71,19 @@ function ConfirmationPage({ data, systemId, facilityDetails, submitStatus }) {
         )}
         <div className="vads-u-margin-top--3 vaos-appts__block-label">
           <AddToCalendar
-            summary={appointmentType.concat(' Appointment')}
+            summary={`Appointment at ${facilityDetails.name}`}
+            description={{
+              text: `You have a health care appointment at ${
+                facilityDetails.name
+              }`,
+              phone: formatFacilityPhone(facilityDetails),
+              additionalText: [
+                'Sign in to VA.gov to get details about this appointment',
+              ],
+            }}
             location={formatFacilityAddress(facilityDetails)}
             startDateTime={data.date1[0]}
+            duration={30}
```

Comment:
Is the default appointment length 30 minutes?

Reply:
I think that is the default, but we should be able to get the duration from the clinic or slot info. I'll make a ticket to update that, 30 minutes is fine for now.